docs/src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -32,6 +32,17 @@ import STYLES from './MarkdownPage.scss';
 
 const getClassName = cssModules(STYLES);
 
+const SEOElements = (props: { description: ?string, title: ?string }) => {
+  const { description, title } = props;
+  return (
+    <Helmet>
+      {title && <title>{title}</title>}
+      {description && <meta name="description" content={description} />}
+      {description && <meta name="og:description" content={description} />}
```

Comment:
I tried doing this:

```jsx
{description && (
 <>
    <meta name="description" content={description} />
    <meta name="og:description" content={description} />
 </>
)}
```

..but it caused an error in `Helmet`, it really didn't like this format. So I went for this way instead.